classes/user/form/ContactForm.inc.php

```diff
@@ -34,11 +34,15 @@ function __construct($user) {
 	 * @copydoc BaseProfileForm::fetch
 	 */
 	function fetch($request, $template = null, $display = false) {
-		$templateMgr = TemplateManager::getManager($request);
 		$site = $request->getSite();
-		$countryDao = DAORegistry::getDAO('CountryDAO'); /* @var $countryDao CountryDAO */
+		$isoCodes = new \Sokil\IsoCodes\IsoCodesFactory();
+		$countries = array();
+		foreach ($isoCodes->getCountries() as $country) {
+			$countries[$country->getAlpha2()] = $country->getLocalName();
+		}
```

Comment:
I notice that here every country results in an array item, but in `api/v1/vocabs/PKPVocabHandler.inc.php` there's code to check if a language has an Alpha2 value:

```php
if (!$language->getAlpha2() || $language->getType() != 'L' || $language->getScope() != 'I') continue;
```

Do these conditionals need to be applied here?

Reply:
No, that's only necessary for the language list, not the country list.